samples/docker-compose.yml

```diff
@@ -10,31 +10,31 @@ services:
     networks:
       - smartclide-monitoring
   # smartclide-monitoring
-  smartclide-monitoring:
-    image: ghcr.io/eclipse-researchlabs/smartclide/smartclide-monitoring:latest
-    depends_on:
-      - rabbitmq
-    volumes:
+#  smartclide-monitoring:
```

Comment:
These changes probably have been committed by accident?

Reply:
yes, fixed it